src/Tribe/Asset/Admin.php

```diff
@@ -1,21 +1,86 @@
 <?php
 
+
 class Tribe__Events__Asset__Admin extends Tribe__Events__Asset__Abstract_Asset {
+
 	public function handle() {
 		$deps = array_merge(
 			$this->deps,
-			array(
-				'jquery',
-				'jquery-ui-datepicker',
-				'jquery-ui-sortable',
-				'tribe-bumpdown',
-				'underscore',
-				'wp-util',
-			)
+				array(
+					'jquery',
+					'jquery-ui-datepicker',
+					'jquery-ui-sortable',
+					'tribe-bumpdown',
+					'underscore',
+					'tribe-events-php-date-formatter',
+					'wp-util',
+				)
 		);
 
 		$path = Tribe__Events__Template_Factory::getMinFile( tribe_events_resource_url( 'events-admin.js' ), true );
 
 		wp_enqueue_script( $this->prefix . '-admin', $path, $deps, $this->filter_js_version(), true );
+
+		$data = array(
+			'date_with_year'    => tribe_get_date_option( 'dateWithYearFormat', Tribe__Date_Utils::DBDATEFORMAT ),
+			'date_no_year'      => tribe_get_date_option( 'dateWithoutYearFormat', Tribe__Date_Utils::DBDATEFORMAT ),
+			'datepicker_format' => Tribe__Date_Utils::datepicker_formats( tribe_get_option( 'datepickerFormat' ) ),
+			'days'              => array(
+				__( 'Sunday', 'the-events-calendar' ),
```

Comment:
You could probably also do something like `$wp_locale->weekday[0]` (to get Monday) etc but I'm not sure that offers a great advantage, also isn't as readable, to me at least. I think I'd stick with just dropping the textdomain.